ansible/group_vars/all/main.yml

```diff
@@ -0,0 +1,5 @@
+repos_dir: /opt/pantheon-repos
+kratos_dir: "{{ repos_dir }}/kratos"
+moirai_dir: "{{ repos_dir }}/moirai"
+dash_dir: "{{ repos_dir }}/dash"
+data_dir: /data
```

Comment:
data goes in its own lvm, which has  been mounted at /data. so no choice about that. we've at least for now decided to have three top-level directories: `/opt/pantheon`, `/opt/pantheon-repos`, and `/opt/pantheon-config`. we can review that, and consider ways to easily put it in a single directory. but I kinda like the three siblings, it keeps things sible from a deployment standpoint.